src/sinks/gcp/chronicle_unstructured.rs

```diff
@@ -131,6 +137,7 @@ pub struct ChronicleUnstructuredConfig {
     /// Chronicle will reject the entry with an error.
     ///
     /// [unstructured_log_types_doc]: https://cloud.google.com/chronicle/docs/ingestion/parser-list/supported-default-parsers
+    #[configurable(metadata(docs::examples = "WINDOWS_DNS", docs::examples = "{{ log_type }}"))]
```

Comment:
Does this template get rendered correctly? I had trouble getting similar things to work in other sinks.

Reply:
I guess we want to show the raw template here. There are a few sources/sinks where we used some type of JS templating language, and it wasn't working correctly.

Reply:
That example in particular is a usage of Vector's [template syntax](https://vector.dev/docs/reference/configuration/template-syntax/) - regrettably shared by a number of other templating languages 😭